tests/nightly/nightly-autorun.sh

```diff
@@ -16,8 +16,10 @@
 
 # Get the latest code from vmware/vic repo
 cd ~/go/src/github.com/vmware/vic
-git fetch
+git clean -fd
+git fetch https://github.com/vmware/vic master
```

Comment:
I was just playing with it, it didn't look like fetch merged in the changes, so it looked to me like we need the pull too.